### PR TITLE
Introduces a configurable switch to globally activate/disactivate mul…

### PR DESCRIPTION
**Pull request by pindara**
…tiple backend sites

Introduces the parameter ‘site_global_switch’ to the thruk.conf file.
Setting the parameter to 1,  activates two icons on the GUI which allows
all configured backend sites to be activated or deactivated with one
click.

The change provides similar functionality to the ‘compact site panel’.
However the solution differs in that the list of configured backend
sites always remains visible, where as on the ‘compact site panel’ ,
only provides a summary description of the total number of active or
disactive sites.

### CODE REVIEW

**Comment by sni** [comment]
Nice idea. For a first look, you commited a few things which probably do not belong to this PR, ex your .gitignore or the bg.png in the themes folder and some changed settings in the thruk.conf.

**Comment by sni** [comment]
Don't worry :-) Then keep the bg.png for now, i didn't try the PR yet, i just had a look at the changed files so far.

File: .gitignore [hunk 1]
```diff
@@ -1,80 +0,0 @@
-META.yml
```
Comment (sni) [inline .gitignore]: now you removed the .gitignore completly

File: lib/Thruk/Config.pm [hunk 2]
```diff
@@ -533,6 +533,7 @@ sub set_default_config {
         'cookie_auth_domain'                => '',
         'perf_bar_mode'                     => 'match',
         'sitepanel'                         => 'auto',
+        'site_global_switch'                => 1,
```
Comment (sni) [inline lib/Thruk/Config.pm]: i wouldn't enable it right from the start, maybe enable it when there are more than 3 backends. It doesn't make sense for a very small number of backends.

File: thruk.conf [hunk 3]
```diff
@@ -977,4 +978,4 @@ cookie_auth_direct_agents         = (wget|curl|python|Go-http-client)
     #hostmenu    = ^Host            # matches all hosts starting with 'Host'
     #demomenu    = ^Demo;           # matches all services on the 'Demo' host
     #demomenu    = ^Test;           # a menu name can be used multiple times
-</action_menu_apply>
+</action_menu_apply>i
```
Comment (sni) [inline thruk.conf]: there is a trailing "i"

Comment (pindara) [reply thruk.conf]: OK to everything; I'll see what I can do. Thanks

Comment (pindara) [reply thruk.conf]: I made and uploaded the changes this morning. Let me know what you think.

**Pull request closed without merging**